autograder/core/tests/test_models/test_ag_test/test_ag_test_command_result.py

```diff
@@ -48,8 +48,9 @@ def test_create_cmd_result_defaults(self):
         self.assertIsNone(cmd_res.stderr_correct)
         self.assertFalse(cmd_res.stdout_truncated)
         self.assertFalse(cmd_res.stderr_truncated)
-        self.assertEqual(cmd_res.partial_credit_points, 0)
-        self.assertEqual(cmd_res.partial_credit_error, ag_models.PartialCreditError.none)
+        self.assertFalse(cmd_res.custom_scoring_used)
+        self.assertEqual(cmd_res.custom_scoring_points, 0)
+        self.assertEqual(cmd_res.custom_scoring_error, ag_models.CustomScoringError.none)
 
     def test_create_cmd_result_no_defaults(self):
```

Comment:
I think past me named this test wrong--could you rename it to `test_cmd_result_to_dict`?